app/models/wakes/location.rb

```diff
@@ -3,7 +3,8 @@ class Wakes::Location < ActiveRecord::Base
   include Wakes::Metrics::GoogleAnalyticsPageviews
   include Wakes::Metrics::Facebook
 
-  validates :path, :format => { :with => %r{\A\/} }, :uniqueness => true
+  validates :path, :format => { :with => %r{\A\/} }
+  validates_uniqueness_of :path, :scope => :host
```

Comment:
I think you can keep the old format, and use `validates :uniqueness => { :scope => :host }`